themes/mongodb/src/package-lock.json

```diff
@@ -1,6 +1,6684 @@
 {
+  "name": "src",
+  "lockfileVersion": 2,
```

Comment:
Just wanted to confirm: Do we want to upgrade the `lockfileVersion`? Those with older versions of `npm` will need to be mindful of this change

Reply:
Unsure, but we've kept the `package-lock` at this version for so long already. 😅 I don't have a strong opinion either but I was a bit concerned about this file bouncing back and forth between versions 1 and 2 in case there were contributors with older versions